integrated_channels/degreed2/migrations/0019_alter_degreed2learnerdatatransmissionaudit_index_together.py

```diff
@@ -38,3 +57,10 @@ class Migration(migrations.Migration):
                 ]
             ),
         ]
+    else:
+        operations = [
+            migrations.AlterIndexTogether(
+                name='degreed2learnerdatatransmissionaudit',
+                index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
+            ),
+        ]
```

Comment:
I'm confused by this. Why is this here? When would this ever execute for us, if it's not sqlite, postgres, or mysql?

Reply:
you are right I might have overthink here I can cover sqlite with else only.